authors/apps/authentication/urls.py

```diff
@@ -6,6 +6,6 @@
 app_name="authentication"
 urlpatterns = [
     path('user/', UserRetrieveUpdateAPIView.as_view()),
-    path('users/', RegistrationAPIView.as_view()),
-    path('users/login/', LoginAPIView.as_view()),
+    path('users/', RegistrationAPIView.as_view(),name="auth-register"),
```

Comment:
Space after comma. Please read the PEP 8 standards on Indentation and implement throughout your code. You can search for a VSCode extension to help you with that.